### ISSUE TITLE
Tuning Hyperparameters

### ISSUE THREAD

**Issue by RavnaBergsndot** [gcp/leela-zero#521]
As far as we know there are a bunch of hyperparameters that could change without re-training the neural network.

1. c_puct

2. virtual loss

3. softmax temperature (changing only the softmax activation part without tampering the rest)

Could someone with a good GPU do some quick experiments to show whether these hyperparameters we are using are indeed close to their optimal values? Currently my computer doesn't have a powerful GPU to do it in a reasonable time.

**Comment by ywrt** [comment]
> virtual loss

Given that by default autogtp runs leelaz with `-t 1`, this doesn't actually do anything. There's no 2nd second running to ever see a node with virtual loss set.

**Comment by sethtroisi** [comment]
> c_puct

GCP tuned this in "Change first-play-urgency. #238" see https://github.com/gcp/leela-zero/pull/238#issuecomment-349576153

**Comment by gcp** [comment]
Softmax may be useful for optimizing gameplay with a finished network but changing it during the learning means the loop where the policy network tries to predict the search output gets distorted: the probability that the learner learns (which assumes softmax temp=1.0) doesn't match the probability that gets fed into the start of the search. So I wouldn't change this from the default as long as we're learning.
